backends/transforms/test/test_remove_clone_ops.py

```diff
@@ -123,6 +140,58 @@ def forward(self, x):
             transformed_gm.code
         )
 
+    def test_clone_channels_last_survives(self):
```

Comment:
maybe should called it `test_clone_non_identity_survives`? cuz it survives because of mutating memory_format / dim_order, rather than cloning a channels_last tensor